assignments/objects.js

```diff
@@ -105,38 +105,41 @@ console.log(intern[4].math(3,4));
 
 // Log the parent object's name
 
-// const parent = {
-//   'name': "Susan",
-//   'age': 70,
-//   speak: function(){
-//     return 'My name is ' + this.name;
-//   }
-//   child{
-//     'name': 'George',
-//     'age': 50,
-//     speak: function(){
-//     return 'My name is ' + this.name;
-//   },
+const parent = {
```

Comment:
Just in case you wanted to see it nested `let parent = {
  'name': "Susan",
  'age': 70,
  'speak': function() {
    return 'My name is ' + this.name
  },
   child: {
    'name': 'George',
    'age': 50,
    'speak': function(){
    return 'My name is ' + this.name
    },
      grandchild: {
        'name': 'George',
        'age': 50,
        'speak': function(){
        return 'My name is ' + this.name
    },
   }
   }
} `

Then to make the child speak would be something like `parent.child.speak()`